lib/cli/interactive-setup/dashboard-login.js

```diff
@@ -17,9 +17,7 @@ const loginOrRegisterQuestion = async (stepHistory, isConsole) =>
 
 const steps = {
   loginOrRegister: async (context) => {
-    const isConsole =
```

Comment:
Thanks for addressing that - I didn't recognize here that its possible to support both integrations